internal/provider/attribute_helpers.go

```diff
@@ -45,9 +43,3 @@ func (c *ConfiguredClient) dataOrDefaultOrganization(ctx context.Context, data A
 
 	return diags
 }
-
-func generateSHA256Hash(data string) string {
-	hasher := sha256.New()
-	hasher.Write([]byte(data))
-	return hex.EncodeToString(hasher.Sum(nil))
-}
```

Comment:
We should leave this in place for now since the `tfe_variable` and `tfe_test_variable` resources still depend on it.